cmd/peer-rest-client.go

```diff
@@ -660,6 +661,20 @@ func (client *peerRESTClient) LoadGroup(group string) error {
 	return nil
 }
 
+// LoadIAMRefreshData - send iam refresh command to peers.
+func (client *peerRESTClient) LoadIAMRefreshData(c *IAMCache) error {
+	b, err := json.Marshal(c)
```

Comment:
Will check about gob.

Reply:
A compressed JSON would be better IMO instead of gob @donatello 

However using JSON in this manner has a higher memory usage. Better to do something like json.NewEncoder() to allow for a more io.Pipe() style usage to avoid a large memory build up.